app/src/test/java/org/fossasia/openevent/app/presenter/EventDetailPresenterTest.java

```diff
@@ -192,4 +192,19 @@ public void shouldNotAccessView() {
 
         verifyZeroInteractions(eventDetailView);
     }
+
+    @Test
+    public void shouldHideProgressbarCorrectly() {
+        when(eventRepository.getAttendees(id, false))
+            .thenReturn(Observable.fromIterable(attendees));
+
+        when(eventRepository.getEvent(id, false))
+            .thenReturn(Observable.just(event));
+
+        InOrder inOrder = Mockito.inOrder(eventRepository, eventDetailView);
```

Comment:
There's no transaction with eventRepository, so please remove it from in order, and move the progress bar showing hiding to single test and remove the show code from above two tests